cooked-validators/src/Cooked/Tx/Constraints/Pretty.hs

```diff
@@ -41,7 +41,7 @@ prettyWallet pkh =
     phash = prettyHash pkh
 
 prettyOutConstraint :: OutConstraint -> Doc ann
-prettyOutConstraint (PaysScript val datum value) =
+prettyOutConstraint (PaysScript val _ datum value) =
```

Comment:
I'm not sure it is worthwhile to hide this when pretty-printing, it will certainly make debugging much more difficult.

Reply:
Yes, I wasn't sure if there was a specific standard for how to pretty print things, I should have mentioned that. Is there a specific way we should print the StakingCredential or does it not matter as long as it's something?

Reply:
It doesn't really matter, as long as its something